apis/kubedb/v1alpha2/constants.go

```diff
@@ -214,32 +214,40 @@ const (
-	PerconaXtraDBInitDBMountPath              = "/docker-entrypoint-initdb.d"
-	PerconaXtraDBCustomConfigMountPath        = "/etc/percona-server.conf.d/"
-	PerconaXtraDBClusterCustomConfigMountPath = "/etc/mysql/custom.conf.d/"
-	PerconaXtraDBCustomConfigVolumeName       = "custom-config"
-	PerconaXtraDBTLSConfigCustom              = "custom"
-	PerconaXtraDBInitContainerName            = "px-init"
-	PerconaXtraDBCoordinatorContainerName     = "px-coordinator"
-	PerconaXtraDBRunScriptVolumeName          = "run-script"
-	PerconaXtraDBRunScriptVolumeMountPath     = "/run-script"
-	PerconaXtraDBInitScriptVolumeName         = "init-scripts"
-	PerconaXtraDBInitScriptVolumeMountPath    = "/scripts"
-	PerconaXtraDBContainerName                = ResourceSingularPerconaXtraDB
+	PerconaXtraDBClusterRecommendedVersion     = "5.7"
+	PerconaXtraDBMaxClusterNameLength          = 32
+	PerconaXtraDBStandaloneReplicas            = 1
+	PerconaXtraDBDefaultClusterSize            = 3
+	PerconaXtraDBDataMountPath                 = "/var/lib/mysql"
+	PerconaXtraDBDataLostFoundPath             = PerconaXtraDBDataMountPath + "/lost+found"
+	PerconaXtraDBInitDBVolumeName              = "initial-script"
+	PerconaXtraDBInitDBMountPath               = "/docker-entrypoint-initdb.d"
+	PerconaXtraDBCustomConfigMountPath         = "/etc/percona-server.conf.d/"
+	PerconaXtraDBClusterCustomConfigMountPath  = "/etc/mysql/custom.conf.d/"
+	PerconaXtraDBCustomConfigVolumeName        = "custom-config"
+	PerconaXtraDBTLSConfigCustom               = "custom"
+	PerconaXtraDBInitContainerName             = "px-init"
+	PerconaXtraDBCoordinatorContainerName      = "px-coordinator"
+	PerconaXtraDBRunScriptVolumeName           = "run-script"
+	PerconaXtraDBRunScriptVolumeMountPath      = "/run-script"
+	PerconaXtraDBInitScriptVolumeName          = "init-scripts"
+	PerconaXtraDBInitScriptVolumeMountPath     = "/scripts"
+	PerconaXtraDBContainerName                 = ResourceSingularPerconaXtraDB
+	PerconaXtraDBCertMountPath                 = "/etc/mysql/certs"
+	PerconaXtraDBExporterConfigFileName        = "exporter.cnf"
+	PerconaXtraDBGaleraClusterPrimaryComponent = "Primary"
+	PerconaXtraDBServerTLSVolumeName           = "tls-server-config"
+	PerconaXtraDBClientTLSVolumeName           = "tls-client-config"
+	PerconaXtraDBExporterTLSVolumeName         = "tls-exporter-config"
+	PerconaXtraDBMetricsExporterTLSVolumeName  = "tls-metrics-exporter-config"
+	PerconaXtraDBMetricsExporterConfigPath     = "/etc/mysql/config/exporter"
 
 	// =========================== MariaDB Constants ============================
 	MariaDBMaxClusterNameLength         = 32
 	MariaDBStandaloneReplicas           = 1
 	MariaDBDefaultClusterSize           = 3
 	MariaDBDataMountPath                = "/var/lib/mysql"
-	MariaDBDataLostFoundPath            = MariaDBDataMountPath + "lost+found"
+	MariaDBDataLostFoundPath            = MariaDBDataMountPath + "/lost+found"
```

Comment:
Use `filepath.Json(MariaDBDataMountPath, "lost+found")`

Reply:
It's giving this error:
```
Const initializer 'filepath.Join()' is not a constant
```
reason: https://stackoverflow.com/questions/46352334/declare-constants-in-golang-using-os-getenv-results-in-const-initializer-in-os